editor.planx.uk/src/pages/Preview/TestWarningPage.tsx

```diff
@@ -0,0 +1,27 @@
+import Box from "@mui/material/Box";
+import Card from "@planx/components/shared/Preview/Card";
+import { CardHeader } from "@planx/components/shared/Preview/CardHeader/CardHeader";
+import React, { PropsWithChildren, useState } from "react";
+
+export const TestWarningPage = ({ children }: PropsWithChildren) => {
+  const [showChildren, setShowChildren] = useState(false);
+  return (
+    <>
+      {showChildren ? (
+        children
+      ) : (
```

Comment:
```suggestion
  const [hasAcknowledgedWarning, setHasAcknowledgedWarning] = useState(false);
  if (hasAcknowledgedWarning) return children;
```

Something like this may be clearer? `showChildren` holds no semantic meaning and is describing the what the behaviour (which can already be understood from the code).